l10n_br_pos/models/res_partner.py

```diff
@@ -32,7 +50,15 @@ def create_from_ui(self, partner):
         from openerp.addons.l10n_br_base.tools import misc
         cnpj_cpf = misc.punctuation_rm(partner['cnpj_cpf'])
         cnpj_cpf_type = 'cpf' if len(cnpj_cpf) == 11 else 'cnpj'
+        partner['data_alteracao'] = fields.Date.today()
```

Comment:
Somente um campo de data de alteração é o suficiente, é identificado qual(is) campo(s) foi alterado(s) nesta última alteração?